zmon_cli/main.py

```diff
@@ -286,6 +286,18 @@ def delete(url):
     return response
 
 
+def delete_body(url, body):
```

Comment:
As mentioned in the Controller PR (https://github.com/zalando/zmon-controller/pull/22/files#diff-03be48f030837aa69830e29d55431360R44) I would expect the "DELETE" to take only a single path argument (not a complete body)